CHANGELOG.md

```diff
@@ -7,8 +7,13 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 - operator-rs: 0.25.0 -> 0.27.1 ([#212]).
+- Shortened the registration socket path for Microk8s compatibility ([#231]).
+  - After upgrading you will need to
+    `rmdir /var/lib/kubelet/plugins_registry/secrets.stackable.tech-reg.sock` manually.
```

Comment:
What happens if you don't?

Reply:
secret-operator will fail to start back up.

Reply:
Oh! Well, then my breaking change label is correct after all.....

Reply:
Sadly, yes.